tests/unittests/tests-flashpage/Makefile.include

```diff
@@ -0,0 +1,5 @@
+# https://github.com/RIOT-OS/RIOT/pull/15859 exposed a bug in the
+# handling of FEATURES_REQUIRED_ANY, so https://github.com/RIOT-OS/RIOT/pull/15935
+# did not run unittests, native is currently failing, so blacklist while this
+# is still the case
+FEATURES_BLACKLIST += arch_native
```

Comment:
```suggestion
FEATURES_REQUIRED += periph_flashpage
```

this does not fail

Reply:
It wouldn't be a unittest anymore then.

Reply:
Anyway I think the requirement makes sense should I push that?

Reply:
`FEATURES_OPTIONAL` would also work, but might raise some eyebrows.

Reply:
I required it, lets fix later if not liked.